nuxeo-coldstorage-web/nuxeo-coldstorage.html

```diff
@@ -20,10 +20,19 @@
   </template>
 </nuxeo-slot-content>
 
+<nuxeo-slot-content name="moveToColdStorageDocumentContentAction" slot="DOCUMENT_ACTIONS" order="121">
+  <template>
+    <nuxeo-delete-contents-from-coldstorage-button document="[[document]]"></nuxeo-delete-contents-from-coldstorage-button
+  </template>
+</nuxeo-slot-content>
+
+
```

Comment:
I can't really judge Web UI code, but I don't think there's a need for two empty lines here.